components/pages/AccountPage/ElementLineCreate.tsx

```diff
@@ -0,0 +1,75 @@
+import FormControl from "@mui/material/FormControl";
+import FormHelperText from "@mui/material/FormHelperText";
+import MenuItem from "@mui/material/MenuItem";
+import Select from "@mui/material/Select";
+import TableCell from "@mui/material/TableCell";
+import TableRow from "@mui/material/TableRow";
+import TextField from "@mui/material/TextField";
+import { Controller, SubmitHandler } from "react-hook-form";
+import { useAccountPage } from "~/components/pages/AccountPage/context";
+import { CreateAccountElementSchemaType, useCreateAccountElementForm } from "~/forms/createAccountElement";
+import { ElementType } from "~/interfaces/accountElement";
+
+export default function ElementLineCreate() {
+  const { saveCreateAccountElement } = useAccountPage();
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    getValues,
+    control,
+    reset,
+  } = useCreateAccountElementForm({});
+
+  const defaultValues = {
+    name: "",
+    type: "",
+  };
+
+  const onSubmit: SubmitHandler<CreateAccountElementSchemaType> = async () => {
+    const values = getValues();
```

Comment:
change this approach to values function props if possible. if not please respond